moderncvbodyvi.sty

```diff
@@ -0,0 +1,180 @@
+%% start of file `moderncvbodyvi.sty'.
```

Comment:
This is a comment in general and does not belong directly to `moderncvbodyvi`. I just did not know how to comment on a file that is not involved in the "files changed" tab ^^. Sry.

It would be nice if we can include your style into the `template.tex` file in order to get a first impression how to use new style. 
In principle I could also add this because it appears to me that there is not really much to do. My suggestions regarding changes for the template are like this:
```diff
diff --git a/template.tex b/template.tex
index 750b4ce..2d9b2a2 100644
--- a/template.tex
+++ b/template.tex
@@ -9,8 +9,8 @@
 \documentclass[11pt,a4paper,sans]{moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')
 
 % moderncv themes
-\moderncvstyle{classic}                            % style options are 'casual' (default), 'classic', 'banking', 'oldstyle' and 'fancy'
-\moderncvcolor{blue}                               % color options 'black', 'blue' (default), 'burgundy', 'green', 'grey', 'orange', 'purple' and 'red'
+\moderncvstyle{contemporary}                       % style options are 'casual' (default), 'classic', 'banking', 'oldstyle', 'fancy' and 'contemporary'
+\moderncvcolor{cerulean}                           % color options 'black', 'blue' (default), 'burgundy', 'green', 'grey', 'orange', 'purple' and 'red'; for 'contemporary' style use 'cerulean'
 %\renewcommand{\familydefault}{\sfdefault}         % to set the default font; use '\sfdefault' for the default sans serif font, '\rmdefault' for the default roman one, or any tex font name
 %\nopagenumbers{}                                  % uncomment to suppress automatic page numbering for CVs longer than one page
 
@@ -114,7 +114,7 @@
 %-----       resume       ---------------------------------------------------------
 \makecvtitle
 
-\section{Education}
+\section{Education}  % for 'contemporary' style use optional argument for displaying an icon: \section[<icon>]{Education}
 \cventry{year--year}{Degree}{Institution}{City}{\textit{Grade}}{Description}  % arguments 3 to 6 can be left empty
 \cventry{year--year}{Degree}{Institution}{City}{\textit{Grade}}{Description}
 
@@ -147,7 +147,7 @@ Detailed achievements:
 \subsection{Miscellaneous}
 \cventry{year--year}{Job title}{Employer}{City}{}{Description}
 
-\section{Languages}
+\section[\emailsymbol]{Languages}
 \cvitemwithcomment{Language 1}{Skill level}{Comment}
 \cvitemwithcomment{Language 2}{Skill level}{Comment}
 \cvitemwithcomment{Language 3}{Skill level}{Comment}
```
Once this PR is merged I also can try to add a more detailed explanation in the new user guide.

Reply:
Thanks for the suggestion.  Typically, the theme also looks better when the margins (esp. the top one) are reduced.  I can mention that also in `template.tex`.

Reply:
I have carried out these changes.  However, the theme looks kind-of broken due to the big heading; perhaps we could comment out by default some of the `\social` commands in the template (?).

Also, per the suggested changes above, `contemporary` would become the new default - I'm quite proud of that, haha!